src/models/RevAiApiDeploymentConfigConstants.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-shadow */
```

Comment:
is this necessary?

Reply:
it fails eslint for this export otherwise